lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -219,7 +184,7 @@ export class PublicClientApplication {
 
             if (StringUtils.isEmpty(loginRequestUrl) || loginRequestUrl === "null") {
                 // Redirect to home page if login request url is null (real null or the string null)
-                this.authModule.logger.warning("Unable to get valid login request url from cache, redirecting to home page");
+                console.warn("Unable to get valid login request url from cache, redirecting to home page");
```

Comment:
Why not use `logger`?

Reply:
The authModule no longer exists here, I need to find a way to expose the logger independently of the client.